recipes/meson/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+$PYTHON setup.py install 
```

Comment:
Please use the script subsection, see the [examples](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml)